tldr/cfg_extractor.py

```
@@ -2197,3 +2209,265 @@ def extract_elixir_cfg(source: str, function_name: str) -> CFGInfo:
+        complexity = self.decision_points + 1
+
+        return CFGInfo(
+            function_name=func_name,
+            blocks=self.blocks,
+            edges=self.edges,
+            entry_block_id=self.entry_block_id,
+            exit_block_ids=self.exit_block_ids,
+            cyclomatic_complexity=complexity,
+        )
+
+    def _visit_node(self, node):
```

Ruff (0.14.14)

Missing return type annotation for private function `_visit_node`

Add return type annotation: `None`

(ANN202)